src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -0,0 +1,653 @@
+---
+title: Optimize Largest Contentful Paint
+subhead: How to render your main content faster.
+authors:
+  - houssein
+date: 2020-04-30
+hero: hero.png
+alt: Optimize LCP banner
+description: | 
+  Largest Contentful Paint (LCP) can be used to determine when the main content of the page has
+  finished rendering on the screen. Learn how to optimize LCP by improving slow server response
+  times, resource load times and client-side rendering.
+tags:
+  - post
+  - fast
+  - web-vitals
+  - largest-contenful-paint
+  - lcp
+  - lighthouse
+---
+
+<blockquote>
+  <p>
+    I can't see any useful content! Why does it take so long to load? 😖
+  </p>
+</blockquote>
+
+One factor contributing to a poor user experience is how long it takes a user to see any content
+rendered to the screen. [First Contentful Paint](/fcp) (FCP) measures how long it
+takes for initial DOM content to render, but it does not capture how long it took the largest
+(usually more meaningful) content on the page to render. A large hero image that takes up most of
+the device screen is a good example, and could load much later than early DOM content that renders
+on the page such as text.
+
+[Largest Contentful Paint](/lcp) (LCP) is a newer [Core Web
+Vitals](/metrics) metric and measures when the largest content element in the
+viewport becomes visible. It can be used to determine when the main content of the page has finished
+rendering on the screen.
+
+<img class="w-screenshot w-screenshot--filled" src="lcp-distributions.png" alt="Good LCP values are
```

Comment:
Do you think it makes sense to use the same markup and image I have here? This should be fine as long as my PR is merged first: https://github.com/GoogleChrome/web.dev/pull/2764/files#diff-cef065d40e06c392ef447cfba4a0855aR70-R76

cc: @addyosmani

Reply:
Done. Will check and merge after your PR gets merged